features/persoon/verblijfplaats/adres/test/vastgesteld-verblijft-niet-op-adres-test.feature

```diff
@@ -0,0 +1,66 @@
+#language: nl
+
+@proxy
+Functionaliteit: leveren 'indicatie vastgesteld verblijft niet op adres' veld bij het vragen van verblijfplaats gegevens test
+
+      Achtergrond:
+        Gegeven adres 'A1' heeft de volgende gegevens
+        | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) | identificatiecode verblijfplaats (11.80) | identificatiecode nummeraanduiding (11.90) |
+        | 0519                 | Spui               | 123                | 0599010000208579                         | 0599200000219678                           |
+        En de persoon met burgerservicenummer '000000152' is ingeschreven op adres 'A1' met de volgende gegevens
+        | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum aanvang adreshouding (10.30) | datum ingang geldigheid (85.10) |
+        | 089999                          | 20020701                       | 20000423                           | 20220222                        |
+
+
+  Regel: aanduiding in onderzoek waarde '089999' wordt geleverd als indicatieVastgesteldVerblijftNietOpAdres verblijfplaats veld en alle geleverde verblijfplaats velden zijn in onderzoek
+
+    Abstract Scenario: gevraagde persoon verblijft niet meer op het geregistreerde adres en <groep> veld <veld> wordt gevraagd
```

Comment:
de gevraagde persoon verblijft volgens mij nog op het geregistreerde adres of mis ik hier iets?

Reply:
Ik denk dat ik het nu begrijp. Misschien dekt de volgende scenario titel meer de situatie: vastgesteld gevraagde persoon verblijft niet meer op het actuele adres en <veld> van het actuele adres wordt gevraagd

Reply:
ik voeg het woord "actuele" toe. 

Ik denk dat het wel goed is in de titel het gevraagde veld te benoemen, dus niet alleen vragen van het (actuele) adres. 
In de persoon bevragen API vragen we altijd actuele gegevens, dus daar hoef je denk ik niet te benoemen dat je het actuele adres vraagt, toch?